index.html

```diff
@@ -0,0 +1,98 @@
+      </p>
+        <ul class="intro-list">
+          <li>Client-side JavaScript extends the core language by supplying objects to control a browser and its Document Object Model (DOM). For example, client-side extensions allow an application to place elements on an HTML form and respond to user events such as mouse clicks, form input, and page navigation.</li>
+          <br>
+          <li>Server-side JavaScript extends the core language by supplying objects relevant to running JavaScript on a server. For example, server-side extensions allow an application to communicate with a database, provide continuity of information from one invocation to another of the application, or perform file manipulations on a server.</li>
+        </ul>
+    </section>
+    <section class="main-section" id="What_you_should_already_know">
+      <header>What you should already know</header>
+      <p>
+        This guide assumes you have the following basic background:
+      </p>
+        <ul class="intro-list">
+          <li>A general understanding of the Internet and the World Wide Web (WWW).</li>
+          <br>
+          <li>Good working knowledge of HyperText Markup Language (HTML).</li>
+          <br>
+          <li>Some programming experience. If you are new to programming, try one of the tutorials linked on the main page about JavaScript.</li>
+        </ul>
+    </section>
+    <section class="main-section" id="JavaScript_and_Java">
+      <header>JavaScript and Java</header>
+      <p>JavaScript and Java are similar in some ways but fundamentally different in some others. The JavaScript language resembles Java but does not have Java's static typing and strong type checking. JavaScript follows most Java expression syntax, naming conventions and basic control-flow constructs which was the reason why it was renamed from LiveScript to JavaScript.
+        <br>In contrast to Java's compile-time system of classes built by declarations, JavaScript supports a runtime system based on a small number of data types representing numeric, Boolean, and string values. JavaScript has a prototype-based object model instead of the more common class-based object model. The prototype-based model provides dynamic inheritance; that is, what is inherited can vary for individual objects. JavaScript also supports functions without any special declarative requirements. Functions can be properties of objects, executing as loosely typed methods. <br>
+        JavaScript is a very free-form language compared to Java. You do not have to declare all variables, classes, and methods. You do not have to be concerned with whether methods are public, private, or protected, and you do not have to implement interfaces. Variables, parameters, and function return types are not explicitly typed.</p>
```

Comment:
I think you need to work better on the indentations

Reply:
> I think you need to work better on the indentations

- [ ] You did a great job at using the inline comments, but I see that you didn't explain to the student why should work on the indentation, so please always tell the student the reason for the change
> For example: 😊⤵️
- You can here tell him/her that it will be better 😃 if you work on the indentation because maybe it damages the linter and makes the appearance of writing the code not good, so let's enhance the code by fixing the indentation of the statements  and with that the code will be better and you will be friend for the linter and passing it 😊💯